src/main/java/io/cdap/plugin/dlp/configs/CryptoKeyHelper.java

```diff
@@ -97,11 +100,21 @@ public static void validateKey(FailureCollector collector, String widgetName, Er
           collector.addFailure("Key is a required field for this transform.", "Please provide a value.")
             .withConfigElement(widgetName, gson.toJson(errorConfig));
         } else {
-          if (!(key.length() == 16 || key.length() == 24 || key.length() == 32)) {
+          if (!BaseEncoding.base64().canDecode(key)) {
             errorConfig.setNestedTransformPropertyId("key");
-            collector.addFailure("Key must be 128/192/256 bit.", "Please provide a key of the correct length.")
-              .withConfigElement(widgetName, gson.toJson(errorConfig));
+            collector.addFailure("Key must be base64 encoded.",
+              "").withConfigElement(widgetName, gson.toJson(errorConfig));
```

Comment:
Why the correction is empty string?

Reply:
The correction seems obvious from the error. Do we always need to add a correction string?